### PR TITLE
cursed world gen

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
it works :tm:

### CODE REVIEW

File: mappings/net/minecraft/world/GameRuleDispatcher.mapping [hunk 1]
```diff
@@ -5,7 +5,9 @@ CLASS net/minecraft/class_95 net/minecraft/world/GameRuleDispatcher
 		ARG 1 nbt
 	METHOD method_218 getString (Ljava/lang/String;)Ljava/lang/String;
 		ARG 1 name
-	METHOD method_220 getRule (Ljava/lang/String;Ljava/lang/String;)V
+	METHOD method_220 setRule (Ljava/lang/String;Ljava/lang/String;)V
```
Comment (hYdos) [inline mappings/net/minecraft/world/GameRuleDispatcher.mapping:8]: setGameRule. consistency matters

File: mappings/net/minecraft/world/biome/BiomeDecorator.mapping [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	FIELD field_467 world Lnet/minecraft/class_99;
+	FIELD field_468 random Ljava/util/Random;
+	FIELD field_469 startPos Lnet/minecraft/class_1372;
+	FIELD field_470 worldProperties Lnet/minecraft/class_417;
+	FIELD field_471 clayFeature Lnet/minecraft/class_441;
+	FIELD field_472 sandDiskFeature Lnet/minecraft/class_441;
+	FIELD field_473 gravelDiskFeature Lnet/minecraft/class_441;
+	FIELD field_474 dirtFeature Lnet/minecraft/class_441;
+	FIELD field_475 gravelFeature Lnet/minecraft/class_441;
+	FIELD field_476 graniteFeature Lnet/minecraft/class_441;
```
Comment (hYdos) [inline mappings/net/minecraft/world/biome/BiomeDecorator.mapping:23]: are these not final? if not final these are fine but if they are capitalize

**Review by hYdos — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/biome/BiomeDecorator.mapping (hunk 2 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/world/biome/BiomeDecorator.mapping:23]: they are lateinit instance fields

File: mappings/net/minecraft/world/GameRuleDispatcher.mapping (hunk 1 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/world/GameRuleDispatcher.mapping:8]: concern

**Review by hYdos — APPROVED** [review]

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 3]
```diff
@@ -37,6 +42,7 @@ CLASS net/minecraft/class_1752 net/minecraft/entity/LivingEntity
 	METHOD method_7120 getAbsorption ()F
 	METHOD method_7121 getScoreboardTeam ()Lnet/minecraft/class_655;
 	METHOD method_7125 getRandom ()Ljava/util/Random;
+	METHOD method_7127 getHurtByTimestamp ()I
```
Comment (shedaniel) [inline mappings/net/minecraft/entity/LivingEntity.mapping:45]: getLastHurtTimestamp?

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: fine

File: mappings/net/minecraft/entity/Entity.mapping [hunk 4]
```diff
@@ -94,10 +102,14 @@ CLASS net/minecraft/class_1745 net/minecraft/entity/Entity
...
 	METHOD method_6922 updateFall (DZLnet/minecraft/class_160;Lnet/minecraft/class_1372;)V
 		ARG 1 y
 		ARG 4 block
 		ARG 5 pos
+	METHOD method_6923 setBounds (FF)V
+		ARG 1 width
+		ARG 2 height
 	METHOD method_6927 setWorld (Lnet/minecraft/class_99;)V
 		ARG 1 world
 	METHOD method_6929 setBoundingBox (Lnet/minecraft/class_646;)V
```
Comment (shedaniel) [inline mappings/net/minecraft/entity/Entity.mapping:115]: Might want to change this to setBounds now?

File: mappings/net/minecraft/entity/Entity.mapping [hunk 5]
```diff
@@ -243,18 +266,30 @@ CLASS net/minecraft/class_1745 net/minecraft/entity/Entity
 		ARG 1 player
 	METHOD method_7036 setSwimming (Z)V
 		ARG 1 swimming
+	METHOD method_7037 incrementVelocity (DDD)V
```
Comment (shedaniel) [inline mappings/net/minecraft/entity/Entity.mapping:269]: addVelocity

File: mappings/net/minecraft/entity/Entity.mapping (hunk 4 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/Entity.mapping:115]: what?

Comment (hYdos) [reply mappings/net/minecraft/entity/Entity.mapping:115]: that doesnt feel like a yarn name

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/Entity.mapping:115]: explain what's `feel like a yarn name`

Comment (hYdos) [reply mappings/net/minecraft/entity/Entity.mapping:115]: feels too shortened

**Pull request merged**